### PR TITLE
Clarify expectations around commit patches

### PR DESCRIPTION
**Pull request by D4nte**

### CODE REVIEW

**Comment by D4nte** [comment]
bors merge

**Comment by D4nte** [comment]
FYI @littlealex003. I thought I'd clarify a bit more my expectations :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+We try to keep a number of [`good first issue`](https://github.com/status-im/js-waku/labels/good%20first%20issue) for new contributors.
+If you feel confident, you can also check out the [`help wanted`](https://github.com/status-im/js-waku/labels/help%20wanted) issues.
+
+To have an idea of the work in the pipeline, feel free to view the [js-waku project board](https://github.com/status-im/js-waku/projects/1).
+This project board is to prioritize the work of core contributors so do not be deterred by issues in the `Icebox` or bottom of the `Backlog`.
+
+Do note that we have a [CI](./.github/workflows/ci.yml) powered by GitHub Action.
+To help ensure your PR passes, just run before committing:
+
+ - `npm run fix`: To format your code,
```
Comment (littlealex003) [inline CONTRIBUTING.md:14]: @D4nte should we put these both in a pre-commit hook?

**Comment by littlealex003** [comment]
> FYI @littlealex003. I thought I'd clarify a bit more my expectations :)

Yes thanks, trying to hit the ground running and see how things go.

Certainly has been fun to work on it a bit and I'm trying to figure out exactly how you want me to do things; not sure how helpful I am to you but I'd be happy to find an interesting remote project like this to work on if you want.

File: CONTRIBUTING.md (hunk 1 above)
Comment (D4nte) [reply CONTRIBUTING.md:14]: https://github.com/status-im/js-waku/issues/110

**Pull request merged**